specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/ApplicationGatewayCreate.json

```diff
@@ -180,48 +184,6 @@
             }
           }
         ],
-        "urlPathMaps": [
-          {
-            "name": "pathMap1",
-            "properties": {
-              "defaultBackendAddressPool": {
-                "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendAddressPools/appgwpool"
-              },
-              "defaultBackendHttpSettings": {
-                "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendHttpSettingsCollection/appgwbhs"
-              },
-              "defaultRewriteRuleSet": {
-                "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/rewriteRuleSets/rewriteRuleSet1"
-              },
-              "defaultLoadDistributionPolicy": {
-                "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/loadDistributionPolicies/ldp1"
-              },
-              "pathRules": [
-                {
-                  "name": "apiPaths",
-                  "properties": {
-                    "paths": [
-                      "/api",
-                      "/v1/api"
-                    ],
-                    "backendAddressPool": {
-                      "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendAddressPools/appgwpool"
-                    },
-                    "backendHttpSettings": {
-                      "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendHttpSettingsCollection/appgwbhs"
-                    },
-                    "rewriteRuleSet": {
-                      "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/rewriteRuleSets/rewriteRuleSet1"
-                    },
-                    "loadDistributionPolicy": {
```

Comment:
Can we set the policy like "loadDistributionPolicy":{} instead of removing the entire UrlPathMap?

Reply:
Tested with empty loaddistributionpolicy, which returns error like: 
```
{
  "error": {
    "code": "InvalidRequestFormat",
    "message": "Cannot parse the request.",
    "details": [
      {
        "code": "MissingJsonReferenceId",
        "message": "Value for reference id is missing. Path properties.urlPathMaps[0].properties.defaultLoadDistributionPolicy."
      },
      {
        "code": "MissingJsonReferenceId",
        "message": "Value for reference id is missing. Path properties.urlPathMaps[0].properties.pathRules[0].properties.loadDistributionPolicy."
      }
    ]
  }
}
```